Formula/mongo-c-driver.rb

```diff
@@ -0,0 +1,75 @@
+class MongoCDriver < Formula
+  desc "C driver for MongoDB"
+  homepage "https://github.com/mongodb/mongo-c-driver"
+  url "https://github.com/mongodb/mongo-c-driver/releases/download/1.5.0/mongo-c-driver-1.5.0.tar.gz"
+  sha256 "b9b7514052fe7ec40786d8fc22247890c97d2b322aa38c851bba986654164bd6"
+
+  bottle do
+    cellar :any
+    sha256 "1e3697dc3b2725914b720cb2aff628d1abeccfd26fc74ebe24bf3be83a5b5ba3" => :sierra
+    sha256 "f53836a2f6179cc4e65f6ef78c5a71792f9bd319676e9e8ef06d8fa5b4fafc5e" => :el_capitan
+    sha256 "c9fce4411ff385d090dc9e20529a045fa98b91801edba2732015f674fa581763" => :yosemite
+  end
+
+  mongo_c_driver_version = version
+  resource "libbson" do
+    url "https://github.com/mongodb/libbson/releases/download/#{mongo_c_driver_version}/libbson-#{mongo_c_driver_version}.tar.gz"
+    sha256 "ba49eeebedfc1e403d20abb080f3a67201b799a05f4a012eee94139ad54a6e6f"
+  end
+
+  head do
+    url "https://github.com/mongodb/mongo-c-driver.git"
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+    depends_on "libtool" => :build
+
+    resource "libbson" do
+      url "https://github.com/mongodb/libbson.git"
+    end
+  end
+
+  depends_on "pkg-config" => :build
+  depends_on "openssl" => :recommended
```

Comment:
Will that use OpenSSL or Secure Transport? Homebrew prefers using its own `openssl` instead of the system openssl, because the system openssl library is not updated, which has security implications.